source/administration/command-line-tools.rst

```diff
@@ -73,6 +73,11 @@ On Docker install, the ``/mattermost/bin`` directory was added to ``PATH``, so y
   .. code-block:: bash
 
     docker exec -it <your-mattermost-container-name> mattermost version
+    
+Using the CLI on Docker Preview
+-------------------------------
+
+The preceding documentation and command reference below also applies to the Mattermost docker preview image.
```

Comment:
Propose adding a link to the Docker preview link, e.g.

```
The preceding documentation and command reference below also applies to the `Mattermost docker preview image <https://github.com/mattermost/mattermost-docker-preview>`_.
```

Reply:
I made the changes. I squashed the Mattermost docker preview commits into 
63bdf45. Thanks @jasonblais :)